codes/quantum/qubits/t-designs.yml

```diff
@@ -0,0 +1,91 @@
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: t-designs
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'T-design code'
+
+# First reference with the code (optional).
+introduced: 'See Application III in \cite{doi:10.1007/s00220-016-2706-8}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A state has topological quantum order if there exists an orthogonal state which cannot be distinguished from it by local measurements. In this code, the code states are a pair of approximately topologically ordered states produced by starting with any two orthogonal states on \(n\) qubits and acting with random local circuits of depth \(\leq 10^9 n\).'
+
+protection: 'Protects approximately against a set of errors localized to a region of size \(n/4\) with \(n\) physical qubits. Deviations from ideal quantum error correction conditions are exponentially suppressed in \(n\).'
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+#  rate: 'k/n, usually listed for a family of codes, but can be notable for a single one. Discussed alongside other parameters such as check operator weight, distance, etc.'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders: 
+    - 'random local circuit of depth \(\leq 10^9 n\)'
+
+#  decoders: 
+#    - 'MWPM decoding algorithm \cite{doi:good-paper} with ... overhead.'
+#    - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+  
+
+
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+#realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  #- 'Code used in DVDs \cite{doi:####...}, 5G, etc.'
+  #- 'Realized in trapped-ion quantum devices \cite{arXiv:####.#####}, etc.'
+
+#notes:
+#  - 'Bounds on n,k, or d for this class.'
+
+#relations:
+#  parents:
```

Comment:
@mgullans What parents should go here, if any? Shall we do a "Random code" umbrella entry?

Reply:
The grandparent could be "Dynamically generated quantum codes" with the parent "Random circuit codes".  There are a number of examples of children of the grandparent that are not defined by random circuits.

Reply:
For example, Hamiltonian time evolution could be a child of DG-QECC

Reply:
Do u have an idea of how to narrow down a defn of DG-QECC? I mean, since there exists an encoding circuit for all codes, isn't everything a DGQECC?

Reply:
These are codes whose natural definition is in terms of a many-body scaling limit of a local dynamical process.

Reply:
Ok, will create a stub.

Reply:
Thanks yeah that makes sense.  All quantum LDPC codes satisfy this property, but it is not a useful way to go about constructing them.

Reply:
You got a ref to the LDPC code dynamical process?

Reply:
We have our paper with Vedika https://arxiv.org/abs/2004.09560.  There is also the Floquet codes of Hastings and Haah that found a LDPC code through a carefully designed dynamical process https://arxiv.org/abs/2107.02194